electron/utils.ts

```diff
@@ -577,6 +577,73 @@ function killPattern(pattern: string) {
   return ret
 }
 
+/**
+ * Detects MS Visual C++ Redistributable and prompts for its installation if it's not found
+ * Many games require this while not actually specifying it, so it's good to have
+ *
+ * Only works on Windows of course
+ */
+function detectVCRedist(mainWindow: BrowserWindow) {
+  if (!isWindows) {
+    return
+  }
+
+  const detectedVCRInstallations: string[] = []
+  let stderr = ''
+  const child = spawn('wmic', [
```

Comment:
What is this WMIC? is the same as: https://docs.microsoft.com/en-us/windows/win32/wmisdk/wmic ?
BEcause it says that was deprecated on Win10 and perhaps wont work on Win11.

Reply:
It's still working fine on W11, in fact, the `check-disk-space` library we're using uses it as well  
We could also add [an extra library](https://www.npmjs.com/package/node-wmi) to handle WMI calls, if you want to avoid using wmic

Reply:
Is up to you, if its still working then I am good. But if using PS is more future proof, then would be better to use it.

Reply:
So Lets merge it then and after the beta we can rewrite it.